docs/source/user-guide/vision/supported_tasks_and_formats.rst

```diff
@@ -0,0 +1,124 @@
+.. _supported_tasks_and_formats:
+
+===========================
+Supported Tasks and Formats
+===========================
+
+In order to accurately analyze the images, labels and predictions provided in the
+:doc:`VisionData object </user-guide/vision/vision_data_class>`,
+they must be provided in deepchecks accepted format. In this section we will describe the supported formats
+for each supported task type.
+
+The data formats are validated at the creation of the VisionData object based on the first batch of data. However,
+in addition to the automatic validations, it is also important to make sure visually that the data was provided in the
+correct format via the :func:`head <deepchecks.vision.VisionData.head>` functionality.
+
+
+Common Formats
+==============
+All types of data are loaded in a batch by batch manner meaning that each data input contain
+information about several samples. In order to support this, all data inputs are expected to be provided
+as either an iterable in which each entry is the data for a single sample or as a high dimension array or tensor in
+which the first dimension is the number of samples.
```

Comment:
Suggestion to rephrase:

Batch Format
---------------
As the data is received in batches, deepchecks expects each batch to contain an iterable of length N (which represents the number of samples) for each data type (images, labels and predictions). 
Specifically, the accepted format of a batch is a dictionary with the following keys: 'images', 'labels', 'predictions' and 'image_identifiers', each value is an iterable of objects in the correct format.
Except for 'images', all keys are optional, but some checks won't be able to run without labels or predictions, and some check visualizations won't work without image_identifiers.

Reply:
Not sure what I wrote is correct (the last sentence)